sirepo/sim_data/__init__.py

```diff
@@ -658,6 +696,18 @@ def _organize_example(cls, data):
     def _proprietary_code_rpm(cls):
         return f'{cls.sim_type()}.rpm'
 
+    @classmethod
+    def _put_db_file(cls, file_path, uri):
+        _request(
+            'PUT',
+            cfg.supervisor_db_file_uri + uri,
+            data=pkio.read_binary(file_path),
+        ).raise_for_status()
+
+    @classmethod
+    def _sim_file_uri(cls, basename, data):
+        # TODO(e-carlin): Better abstraction
+        return f'{cls.sim_type()}/{data.models.simulation.simulationId}/{basename}'
```

Comment:
@robnagler the knowledge of the path to the simulation_dir is now known here and in simulation_db. Is that a problem?

Reply:
yes

Reply:
67a6aa10